src/main/kotlin/no/nav/familie/klage/blankett/BlankettService.kt

```diff
@@ -38,6 +42,15 @@ class BlankettService(
         return blankettClient.genererBlankett(blankettPdfRequest)
     }
 
+    private fun mapPåklagetVedtak(behandling: BehandlingDto): BlankettPåklagetVedtakDto? {
+        return behandling.påklagetVedtak.eksternFagsystemBehandlingId?.let { påklagetBehandlingId ->
+            val fagsystemVedtak = fagsystemVedtakService.hentFagsystemVedtak(behandling.id)
+            val påklagetVedtak = fagsystemVedtak.singleOrNull { it.eksternBehandlingId == påklagetBehandlingId }
+                ?: error("Finner ikke fagsystemvedtak med eksternBehandlingId=$påklagetBehandlingId")
```

Comment:
kunne `singleOrNull(...)` blitt byttet ut med `first(...)` ? Eller blir det for utydelig feilmelding, tenker du?

Reply:
Tenkte først på `firstOrNull` men tenkte det var like bra å bare ta single då det kun burde finnes en 😬 
Uten `orNull` blir utydelig feilmelding